webservices/resources/reports.py

```diff
@@ -128,7 +128,7 @@ def get_match_filters():
     tags=['financial'],
     description=docs.REPORTS,
     params={
-        'committee_type': {
+        'entity_type': {
             'description': 'House, Senate, presidential, independent expenditure only',
```

Comment:
Please update the description to include pac & party and add the description to docs.py.
<img width="548" alt="Screen Shot 2021-06-21 at 3 25 58 PM" src="https://user-images.githubusercontent.com/24395751/122817576-142f9980-d2a6-11eb-9a26-c0da6f0a9515.png">